apps/modernization-ui/src/apps/page-builder/components/EditPageHeader/EditPageHeader.tsx

```diff
@@ -32,7 +33,7 @@ export const EditPageHeader = ({ page }: PageProps) => {
                         </Button>
                     </>
                 ) : (
-                    <Button type="button" outline>
+                    <Button type="button" outline onClick={() => handleSaveDraft()}>
```

Comment:
You dont need the call back if you're not passing arguements, you can just do `onClick={handleSaveDraft}`